app/src/main/java/mobile/programming/musicapp/MainActivity.kt

```diff
@@ -11,27 +11,22 @@ class MainActivity : AppCompatActivity() {
     }
 }
 
-FirebaseInstanceId.getInstance().instanceId
-        .addOnCompleteListener(OnCompleteListener { task ->
-            if (!task.isSuccessful) {
-                Log.w(TAG, "getInstanceId failed", task.exception)
-                return@OnCompleteListener
-            }
-
-            // Get new Instance ID token
-            val token = task.result?.token
-
-            // Log and toast
-            val msg = getString(R.string.msg_token_fmt, token)
-            Log.d(TAG, msg)
-            Toast.makeText(baseContext, msg, Toast.LENGTH_SHORT).show()
-        })
-
-val fm = FirebaseMessaging.getInstance()
-fm.send(RemoteMessage.Builder("$SENDER_ID@fcm.googleapis.com")
-        .setMessageId(Integer.toString(messageId))
-        .addData("my_message", "Hello World")
-        .addData("my_action", "SAY_HELLO")
-        .build())
-
-FirebaseMessaging.getInstance().isAutoInitEnabled = true
+//FirebaseInstanceId.getInstance().instanceId.addOnCompleteListener(OnCompleteListener {task ->
+//    if (!task.isSuccessful) {
+//        Log.w(TAG, "getInstanceId failed", task.exception)
+//        return@OnCompleteListener
```

Comment:
Commented out this section of MainActivity to eliminate the build errors that were occurring.

Reply:
Avoid committing commented out code to version control.  This is a no-no, as version control offers better alternatives.
- You're in your own branch, so you can do what you want to do without impacting master.  This gives you a license to experiment.  Instead of commenting out code, delete it.
- You can always go back in history to see what used to be there.